obal/data/modules/verify_tag.py

```diff
@@ -0,0 +1,62 @@
+"""
+Verify packages against a tag in Koji
+"""
+
+import os
+from subprocess import STDOUT, check_output
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.obal import get_specfile_name
+
+def main():
+    """
+    Verify a tag
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            packages=dict(type='dict', required=True),
+            tag=dict(type='str', required=True),
+            directory=dict(type='str', required=True)
+        )
+    )
+
+    tag = module.params['tag']
+    packages = module.params['packages']
+    directory = module.params['directory']
+
+    command = ['koji', 'list-pkgs', '--quiet', '--tag', tag]
+
+    packages_for_tag = []
+
+    for (package, attributes) in packages.items():
+        if 'koji_tags' in attributes:
+            if tag in attributes['koji_tags'].keys():
```

Comment:
Did you mean a for loop here?
```suggestion
            for tag in attributes['koji_tags'].keys():
```